aws/diff_suppress_funcs.go

```diff
@@ -8,7 +8,7 @@ import (
 	"strings"
 
 	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
-	"github.com/jen20/awspolicyequivalence"
+	awspolicy "github.com/jen20/awspolicyequivalence"
```

Comment:
Not sure why this appeared here, but the tests pass and it seems like it should be this way... Happy to remove it if it's not needed. (FWIW, I'm relatively new to golang)